certified-connectors/Inserve/apiDefinition.swagger.json

```diff
@@ -0,0 +1,392 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "version": "1.0.0",
+      "title": "Inserve",
+      "description": "Inserve connector to create tickets within our service management software.",
+      "contact": {
+        "name": "Frank Derks",

```

Comment:
@frank-inserve since you confirmed that you are a verified publisher. please change this to the company name.